galaxy_api/api/exceptions.py

```diff
@@ -49,3 +40,23 @@ def exception_handler(exc, context):
     errors = _get_errors(exc.detail, status=exc.status_code, title=title)
     data = {'errors': list(errors)}
     return Response(data, status=exc.status_code, headers=headers)
+
+
+def _handle_api_client_exception(exc):
+    return HttpResponse(exc.body, status=exc.status, content_type=exc.headers['Content-Type'])
```

Comment:
For requests to pulp_ansible that return the JSONApi style errors, this should work okay.

Not sure if there are other pulp_ansible endpoints that may return different formatted error objects. For galaxy-api endpoints that 'wrap' pulp_ansible endpoints via galaxy_pulp, that seems like it could result in clients getting error response bodies in a format they don't know how to deal with.

But I wouldn't try to transform the error objects for those potential cases at this point though. But not worried about it at this point.

In the future, maybe a dedicated exception type for wrapping the exceptions from galaxy_pulp (especially the non JSONApi style).